test/jasmine/tests/config_test.js

```diff
@@ -232,23 +232,25 @@ describe('config argument', function() {
         afterEach(destroyGraphDiv);
 
         it('should not display the edit link by default', function() {
-            Plotly.plot(gd, [], {});
-
-            var link = document.getElementsByClassName('js-plot-link-container')[0];
+            Plotly.plot(gd, [], {})
+            .then(function() {
+                var link = document.getElementsByClassName('js-plot-link-container')[0];
 
-            expect(link).toBeUndefined();
+                expect(link).toBeUndefined();
+            });
         });
 
         it('should display a link when true', function() {
-            Plotly.plot(gd, [], {}, { showLink: true });
-
-            var link = document.getElementsByClassName('js-plot-link-container')[0];
+            Plotly.plot(gd, [], {}, { showLink: true })
+            .then(function() {
```

Comment:
This test and the one above need `done` (and `failTest`) now that their assertions are async.

Reply:
Good eye! Done in https://github.com/plotly/plotly.js/pull/5340/commits/7f9073228128857f340dbd8180260aabbe8f7179.